.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @Azure/azure-osconfig-core @MariusNi @robertschaedler3
+* @Azure/osconfig-maintainers
```

Comment:
>osconfig-maintainer [](http://example.com/codeflow?start=9&length=19)

Why not naming this 'osconfig-core' as it was, rather than osconfig-maintainers which does not truly reflect the scope of this group?

How about:

@Azure/osconfig-core @MariusNi @robertschaedler3

So that way the two real maintainers are always explicitly listed for every PR. Or, if you want to skip yours:

@Azure/osconfig-core @MariusNi

Reply:
The `osconfig-maintainers` name correlates to the GitHub role assignment given to the team. Similar to the `osconfig-admins` team receiving the `admin` role.

https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization

Reply:
This has the same effect whether members are listed explicitly or not. 2 members of the team must review every pull request. The reason to use teams instead of listing all the members explicitly in this file is to prevent needing to change the file if the team structure changes. 

All members of the team will receive email notifications when a PR is opened or when the team is mentioned. It is misleading to have 2 individuals listed there when we are ultimately looking for reviews from team members.